apps/fluent-tester/src/FluentTester/TestComponents/Test.tsx

```diff
@@ -54,6 +54,8 @@ const styles = StyleSheet.create({
   },
 });
 
+const isMobile = Platform.OS == 'android';
```

Comment:
Can you add iOS to this as well? Or just make an issue tracking adding iOS here as well and add the comment so that I or someone else can do that?

Reply:
added check for iOS (excluded iPad)